ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -428,8 +418,10 @@ open class DrawerController: UIViewController {
     private var useCustomBackgroundColor: Bool = false
     /// for iPad split mode, navigation bar has a different dark elevated color, and if it is a `.down` presentation style, match `Colors.NavigationBar.background` elevated color
     private var useNavigationBarBackgroundColor: Bool = false
+    /// token to modify drawer apperance
+    private static let drawerToken = DrawerTokens()
```

Comment:
"drawerTokens" instead? (I see this as a group of design tokens applicable to the Drawer)